tetris.js

```diff
@@ -10,6 +10,17 @@ module.exports = {
 		let a = -1;
 		let localTetromino = []
 		
+		//check that the field is free to rotate	
+		function checkThatTheFieldIsFree(elem){
+			for(let i of elem){
+				//undefined - if this part of the field is missing
```

Comment:
explain further, in which case the field will be missing

Reply:
During tests, if at turning the tetramino will go out of the field

Reply:
Why not check for negative coordinate instead?

Reply:
line 421 in file test-tetris.js have test, L-tetromino too low to turn.  
When tetramino rotates and  it cannot turn because the field is absent, then the field along the field[i.x] axis or field[i.y] axis === undefined.

Reply:
1. Rewrite allItemsOnTheField so it would return boolean (true or false).
2. Rewrite checkThatTheFieldIsFree to use allItemsOnTheField to check if tetramino is on the field